doc/classes/Joint2D.xml

```diff
@@ -74,12 +74,16 @@
 	</methods>
 	<members>
 		<member name="bias" type="float" setter="set_bias" getter="get_bias">
+			The force with which nodes A and B stay in positional relation to each other.
```

Comment:
This needs more description. Is the force constant or proportional to the distance? "positional relation" sounds awkward, how about "the two nodes' relative positions"?